apps/server/src/modules/tool/service/context-external-tool.service.spec.ts

```diff
@@ -80,6 +80,7 @@ describe('ContextExternalToolService', () => {
 				const contextExternalTool2: ContextExternalToolDO = contextExternalToolDOFactory
 					.withSchoolExternalToolRef(schoolExternalToolId)
 					.buildWithId();
+				contextExternalToolRepo.find.mockResolvedValue([contextExternalTool1, contextExternalTool2]);
```

Comment:
Why two times the same mock? We should use the functions with "once" at the end.  mock...Once().